novel.sql

```diff
@@ -0,0 +1,147 @@
+CREATE DATABASE novels;
```

Comment:
You can delete this. We only used .sql  files as a stand in until we learned migrations. Real world projects would use migrations from the beginning and never have these files.